goleft_functions.wdl

```diff
@@ -78,24 +84,26 @@ task indexcovCRAM {
 			fi
 
 			INPUTCRAI=$(echo ~{inputCram}.crai)
-			mkdir indexDir
-			ln -s ~{inputCram} indexDir~{basename(inputCram)}
-			ln -s ${INPUTCRAI} indexDir~{basename(inputCram)}.crai
-			goleft indexcov --extranormalize -d indexDir/ --fai ~{refGenomeIndex} ~{inputCram}.crai
+			mkdir ~{prefix}_indexDir
+			ln -s ~{inputCram} ~{prefix}_indexDir~{cramBasename}
+			ln -s ${INPUTCRAI} ~{prefix}_indexDir~{cramBasename}.crai
```

Comment:
There don't need to be slashes in this?  (as in: `~{prefix}_indexDir/~{cramBasename}.crai`)

Reply:
It doesn't, for some reason. Which has me wondering if the softlinks are needed at all. I'll open an issue about this and do some testing later.